craft_providers/actions/snap_installer.py

```diff
@@ -299,9 +300,11 @@ def _add_assertions_from_host(executor: Executor, snap_name: str) -> None:
             details="error copying snap assert file into target environment",
         ) from error
 
+    cmd = snap_cmd.formulate_ack_command(snap_assert_path=target_assert_path)
+    logger.debug("Executing command: %s", cmd)
```

Comment:
Doesn't this already get logged?

https://github.com/canonical/craft-providers/blob/4c400b1f655755105aa28bdb943dd865482e77c2/craft_providers/lxd/lxc.py#L379C9-L379C74
https://github.com/canonical/craft-providers/blob/main/craft_providers/multipass/multipass.py#L121C2-L121C2

Reply:
Oh, :)